test/dynamo/test_error_messages.py

```diff
@@ -91,29 +91,6 @@ def fn():
     return torch.linalg.lstsq(torch.rand(10, 10), torch.rand(10, 10))""",
             )
 
-    def test_data_dependent_operator(self):
```

Comment:
this test is no longer relevant since we don't expect to ever throw an error anymore. in the case of fullgraph=False we graph break (but don't error, which is what this test suite is for)